knowledge-base/optimize-and-reduce-pdf-size-radpdfprocessing.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Optimizing and Reducing the Size of PDF Files with RadPdfProcessing
+description: Learn how to optimize and reduce the size of PDF files using RadPdfProcessing by applying compression and image quality settings.
```

Comment:
Learn how to optimize and reduce the size of PDF files through the compression and image quality settings when using RadPdfProcessing.